colin-api/Makefile

```diff
@@ -37,7 +37,7 @@ clean-test: ## clean test files
 	rm -fr htmlcov/
 
 build-req: clean ## Upgrade requirements
-	test -f venv/bin/activate || python3.8 -m venv  $(CURRENT_ABS_DIR)/venv ;\
```

Comment:
Checking with the team, but the CI maintainers want it at 3.8

Reply:
ok, will revert

Reply:
reverted